cmd/podman/root.go

```diff
@@ -421,6 +421,15 @@ func persistentPostRunE(cmd *cobra.Command, args []string) error {
 
 func configHook() {
 	if dockerConfig != "" {
+		statInfo, err := os.Stat(dockerConfig)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Supplied --config folder (%s) does exist or is not accessible: %s", dockerConfig, err.Error())
+			os.Exit(1)
+		}
+		if !statInfo.IsDir() {
+			fmt.Fprintf(os.Stderr, "Supplied --config file (%s) is not a directory", dockerConfig)
+			os.Exit(1)
+		}
```

Comment:
considering that breaks the system test I guess it is best to not do that, https://github.com/containers/podman/blob/1b9f78a33c9eab2956963f72f2eb31dba6d85516/test/system/150-login.bats#L94

I assume there is no actual requirement that this path must exist before?

Reply:
Oh so some of the code expects to be able to create the file?!
If this is an assumption we make ... should we remove the failure when things dont exist ... but keep the failure when you supply a regular file?